sdk/resourcemanager/azure-resourcemanager-appplatform/src/main/java/com/azure/resourcemanager/appplatform/models/SpringAppDeployment.java

```diff
@@ -97,6 +100,16 @@ interface DefinitionStages {
              */
             T withJarFile(File jar);
 
+            /**
+             * (Enterprise Tier Only)
+             * Specifies the jar package for the deployment.
+             * @param jar the file of the jar
+             * @param configFilePatterns Config file patterns to decide which patterns of Application Configuration Service will be used.
+             *                           Use null or empty list to clear existing configurations.
+             * @return the next stage of deployment definition
+             */
+            T withJarFile(File jar, List<String> configFilePatterns);
```

Comment:
if `configFilePatterns` is not empty, maybe bind the configuration service for the user?

Reply:
Currently by default, when creating Spring Service, we create the default Configuration Service for the user(this is like implicit `SpringService.WithConfigurationService`). 
If the user wants to use the Configuration Service for their Spring Apps, they will also need to bind their Apps to the existing Configuration Service. I'm thinking that if they create the Apps with `configFilePatterns`, it may suggest this intention and we can bind the App for them.

Reply:
SpingCloudApp.WithConfigurationServiceBinding

Emm, probably no need. Just make it clear in description first.

Reply:
OK, I'll add javadoc on this.